src/main/java/seedu/address/logic/wanderlustlogic/wanderlustparser/WanderlustShowCommandParser.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.wanderlustlogic.wanderlustparser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.wanderlustlogic.wanderlustcommands.ShowCommand;
+import seedu.address.logic.wanderlustlogic.wanderlustparser.exceptions.ParseException;
+
+public class WanderlustShowCommandParser implements WanderlustParserInterface<ShowCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the ShowCommand.
+     * and returns a ShowCommand object for execution.
+     * @throws seedu.address.logic.parser.exceptions.ParseException if the user input does not conform
```

Comment:
should we use {@code ParseException} here instead of seedu.address.logic.parser.exceptions.ParseException?

Reply:
Resolved